components/job-orchestration/job_orchestration/executor/query/fs_search_task.py

```diff
@@ -129,19 +190,11 @@ def search(
             start_time=start_time,
         )
 
-    if worker_config.archive_output.storage.type == StorageType.S3:
-        logger.error(f"Search is not supported for the S3 storage type")
-        return report_task_failure(
-            sql_adapter=sql_adapter,
-            task_id=task_id,
-            start_time=start_time,
-        )
-
     # Make task_command
     clp_home = Path(os.getenv("CLP_HOME"))
     search_config = SearchJobConfig.parse_obj(job_config)
 
-    task_command = make_command(
+    task_command, env_vars = make_command(
```

Comment:
nit: how about changing the name to "core_clp_env_vars"/"query_env_var" or something similar to indicate that this env_var only applies to clp/clp-s executable